pkg/bloomgateway/bloomgateway.go

```diff
@@ -195,7 +195,7 @@ func New(cfg Config, schemaCfg config.SchemaConfig, storageCfg storage.Config, s
 		pendingTasks: makePendingTasks(pendingTasksInitialCap),
 	}
 
-	g.queueMetrics = queue.NewMetrics("bloom_gateway", reg)
+	g.queueMetrics = queue.NewMetrics("bloom_gateway", reg, "loki")
```

Comment:
I think we should have this "loki" string in a constant somewhere. Same for the "cortex" string.

Reply:
Constants were added. It does look kinda busy though so please have a look.